elfpy/utils/apeworx_integrations.py

```diff
@@ -84,6 +90,117 @@ class DefaultHyperdriveConfig:
     target_liquidity = FixedPoint(1 * 10**6)
 
 
+def inspect_dump(trade_history, agent_addresses, hyperdrive_instance, base_instance, tolerance=1e16):
```

Comment:
Can you move both of these into evm_bots and refrain from adding functions to apeworx_integrations? It'll help with my tracking the changes that need to  be made.

Reply:
done in bda60758a4feb8e5df6547ebb6cf820f52a5ec1f